src/Symfony/Bundle/SecurityBundle/Resources/config/security_listeners.xml

```diff
@@ -210,6 +210,7 @@
             <argument /> <!-- Simple Authenticator -->
             <argument /> <!-- User Provider -->
             <argument /> <!-- Provider-shared Key -->
+            <argument>null</argument> <!-- UserChecker -->
```

Comment:
do we need `null`? just wondering as its not consistent with the arguments above :smile:

Reply:
Ah I see @nicolas-grekas added it originally: https://github.com/symfony/symfony/pull/26762#discussion_r178785206

Still needed now with your fix?

Reply:
It is not needed for the fix, but it would be inconsistent not to add it since it is a default parameter.